lib/ansible/utils/module_docs_fragments/fortios_api_doc.py

```diff
@@ -0,0 +1,112 @@
+#
+# (c) 2017, Will Wagner <willwagner602@gmail.com> and Eugene Opredelennov <eoprede@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+class ModuleDocFragment(object):
+
+    # Standard files documentation fragment
+    DOCUMENTATION = """
+notes:
+  - Due to the volume of available options, it is unreasonable to list all the possible configuration options for each module.
+    Instead, run the module with "print_current_config = true" to get full list of possible options output as a json file at
+    the path where the playbook is executed. Additionally, default objects and argument specifications (including options,
+    suboptions, and expected values) are saved locally wherever the playbook is executed.
+  - Module is written to take input of a desired end config of the device. As such, if an object in the configuration
+    is missing from that file, it will be DELETED from the device. It is recommended to use print_current_config = true
+    for onboarding devices with existing configuration.
+  - The module determines whether an existing configuration matches the proposed configuration by checking each key - value
+    pair in the proposal against the existing configuration. Any keys not found in the proposed configuration are assumed
+    to be in their desired state, because the list of keys is exhaustive and the API always returns all keys.
+options:
+  conn_params:
+    description:
+      - Dictionary with all the connection parameters.
+    required: true
+    type: dict
+    suboptions:
+      fortigate_username:
+        description:
+          - Specifies username for Fortigate connection.
```

Comment:
Does Fortigate support different levels of user privileges? If so please state what privileges this user needs to have. 
e.g "Specifies username for Fortigate connection with admin privileges"